Lib/unittest/test/test_case.py

```diff
@@ -425,6 +425,15 @@ def test_c(self):
         expected = ['a1', 'a2', 'b1']
         self.assertEqual(events, expected)
 
+    def test_subtests_debug(self):
+        # Ensure that subTest() does not crash debug() calls (issue #34900)
+        class Foo(unittest.TestCase):
+            def test_a(self):
+                with self.subTest():
+                    pass
```

Comment:
It would be nice to test something here:

```py
events = []

...

def test_a(self):
    events.append('test case')
    with self.subTest():
        events.append('subtest')

...

Foo('test_a').debug()

self.assertEqual(events, [...])
```

Reply:
Sure, done!